src/style.css

```diff
@@ -0,0 +1,82 @@
+body {
+  font-weight: 300;
+}
```

Comment:
add `font-weight` for html and also add `font-family`
move this selector under `@font-face`